<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCreateMap</class>
 <widget class="QWidget" name="NewCreateMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewCreateMap</string>
  </property>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>create map</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>width</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_width">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_height">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>height</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_help">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>help</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
